app/Services/UpdateUserService.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace App\Services;
+
+use Illuminate\Support\Facades\Auth;
+
+class UpdateUserService
+{
+    public function updateUser(array $userData)
+    {
+        $user = Auth::user();
```

Comment:
remove this line. 
pass user as an argument: public function updateUser(User $user, array $userData)...